File: CountriesFlagsExampleObjC/CountriesFlagsExampleObjC/ViewController.m
//
//  ViewController.m
//  CountriesFlagsExampleObjC
//

#import "ViewController.h"

@import CountriesFlagsObjC;

@interface ViewController ()

@property (weak, nonatomic) IBOutlet UIImageView *imageView;

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    
    self.imageView.image = [UIImage flagForCountry:CountriesFlagsUkraine
                                              size:self.imageView.frame.size];
}


@end

File: CountriesFlagsExample/CountriesFlagsExample/ViewController.swift
//
//  ViewController.swift
//  CountriesFlagsExample
//

import UIKit
import CountriesFlags

class ViewController: UIViewController {

    @IBOutlet weak var imageView: UIImageView!

    override func viewDidLoad() {
        super.viewDidLoad()

        imageView.image = UIImage.flag(for: .Ukraine, with: imageView.frame.size)
    }
}
